packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## v7.1.0-dev (July 22, 2025)
 - Add support for environment level base paths on /mobify routes [#2892](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2892)
+- Fix private client endpoint prop name [#3177](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/3177)
```

Comment:
To be clearer and searchable, can we mention the exact change here? Something like "from someProp to newProp".

Reply:
Since this fix is tied to the to-be-released base paths changes, I might just omit this specific entry. Or change it to focus more on the useDNT changes (since those affect parts that have already been released prior).

I'll make this change when I clean up change logs as part of the release process.